tests/unit/fem/test_transfer.cpp

```diff
@@ -199,6 +231,252 @@ TEST_CASE("transfer")
+               {
+                  c_fec = new ND_FECollection(order+1, dimension);
+                  f_fec = new ND_FECollection(order+1, dimension);
+               }
+               else
+               {
+                  c_fec = new RT_FECollection(order, dimension);
+                  f_fec = new RT_FECollection(order, dimension);
+               }
+
+               mesh.EnsureNCMesh();
+               mesh.RandomRefinement(0.5);
+
+               int spaceDimension = 1;
+
+               if (vectorspace == 1)
+               {
+                  spaceDimension = dimension;
+               }
+
+               FiniteElementSpace* c_fespace =
+                  new FiniteElementSpace(&mesh, c_fec, spaceDimension);
+               FiniteElementSpace* f_fespace =
+                  new FiniteElementSpace(&mesh, f_fec,spaceDimension);
+               int maxorder = RandomPRefinement(*f_fespace);
+
+               std::cout  << "  Max fine order: " << maxorder << "\n";
+
+               Operator* referenceOperator = nullptr;
+
+               referenceOperator = new PRefinementTransferOperator(*c_fespace,
+                                                                   *f_fespace);
+
+               TransferOperator testTransferOperator(*c_fespace, *f_fespace);
+               GridFunction X(c_fespace); X = 0.;
+               GridFunction X_cmp(c_fespace); X_cmp = 0.;
+               GridFunction Y_exact(f_fespace); Y_exact = 0.;
+               GridFunction Y_std(f_fespace); Y_std = 0.;
+               GridFunction Y_test(f_fespace); Y_test = 0.;
+               coeff_order = std::min(2,order);
+               if (vectorspace == 0)
+               {
+                  FunctionCoefficient funcCoeff(&coeff);
+                  X.ProjectCoefficient(funcCoeff);
+                  Y_exact.ProjectCoefficient(funcCoeff);
+               }
+               else
+               {
+                  VectorFunctionCoefficient funcCoeff(dimension, &vectorcoeff);
+                  X.ProjectCoefficient(funcCoeff);
+                  Y_exact.ProjectCoefficient(funcCoeff);
+               }
+
+               Y_std = 0.0;
+               Y_test = 0.0;
+
+               referenceOperator->Mult(X, Y_std);
+               Y_std -= Y_exact;
+               REQUIRE(Y_std.Norml2() < 1e-12 * Y_exact.Norml2());
+
+               testTransferOperator.Mult(X, Y_test);
+
+               Y_test -= Y_exact;
+               REQUIRE(Y_test.Norml2() < 1e-12 * Y_exact.Norml2());
+
+               referenceOperator->MultTranspose(Y_exact, X);
+               testTransferOperator.MultTranspose(Y_exact, X_cmp);
+
+               X -= X_cmp;
+               REQUIRE(X.Norml2() < 1e-12 * X_cmp.Norml2());
+
+               delete referenceOperator;
+               delete f_fespace;
+               delete c_fespace;
+               delete f_fec;
+               delete c_fec;
+            }
+         }
+      }
+   }
+}
+
+TEST_CASE("variable_order_true_transfer")
```

Comment:
Should this be a parallel test, since it is for true transfer (a parallel concept)?

Reply:
This is for `TrueDofs` for the AMR serial case. There is only serial support right now for variable order spaces